### ISSUE TITLE
Traditional Chinese(繁體中文) documents have translation errors

### ISSUE THREAD

**Issue by sing-gor** [elixirschool/elixirschool#1967]
In the **Anonymous Functions** .  
```markdown
正如函數名稱所暗示的，匿名函數沒有名字。 同時與我們在 Enum 課程中看到的那樣，匿名函數經常被傳遞給其他函數。 為了在 Elixir 中定義**一個名函數**，我們需要 fn 和 end 來做為關鍵字。 在這兩個關鍵字之間，我們可以定義任意數量由 -> 分隔的參數 (parameters) 和函數主體 (bodies)。
```
To   
```markdown
正如函數名稱所暗示的，匿名函數沒有名字。 同時與我們在 Enum 課程中看到的那樣，匿名函數經常被傳遞給其他函數。 為了在 Elixir 中定義**一個匿名函數**，我們需要 fn 和 end 來做為關鍵字。 在這兩個關鍵字之間，我們可以定義任意數量由 -> 分隔的參數 (parameters) 和函數主體 (bodies)。
```

**Comment by tajchumber** [comment]
@sing-gor we're happy to accept pull requests for this.
Are you able to create one? :)

**Comment by lithiumpie** [comment]
I will fix the typo #1981 Thank you for your contribution @sing-gor 👍

**Comment by doomspork** [comment]
This has been fixed. Thank you everyone 🎉
